tikv-client/src/main/java/com/pingcap/tikv/meta/TiDAGRequest.java

```diff
@@ -823,6 +823,7 @@ public String toString() {
       sb.append(", Limit: ");
       sb.append("[").append(limit).append("]");
     }
+    sb.append(", startTs: ").append(startTs.getVersion());
```

Comment:
I added a startTs information in `TiDagRequest` output. Not really sure if it is useful? @zhexuany

Reply:
It is better to print it out.